test/evergreen.yml

```diff
@@ -60,7 +60,14 @@ functions:
       script: |
         set -o errexit
         set -o verbose
-        git clone https://github.com/mongodb/mongo
+        mongo_repo=https://github.com/mongodb/mongo
+        if [ "${branch_name}" = "mongodb-5.0" ]; then
+          git clone $mongo_repo -b v5.0
+        elif [ "${branch_name}" = "mongodb-4.4" ]; then
+          git clone $mongo_repo -b v4.4
+        else
+          git clone $mongo_repo
```

Comment:
Should we precise the `master` branch?

Reply:
Updated in [a6bf607](https://github.com/wiredtiger/wiredtiger/pull/7603/commits/a6bf607d7d1052c66358e51f4853ecf64d2f92db)